src/pages/Explorer/Category/ExplorerCategory.svelte

```diff
@@ -79,45 +79,49 @@
     items = items
   })
 
-  function fetch(bypassLoading = false) {
+  async function setInsightItems() {
+    if (activeMenu !== MenuItem.TRENDING) return
+    const insightItems = await queryExplorerItems({
+      types: [EntityKeys.INSIGHT],
+      page: insightsPage,
+    })
+    insightsPages = insightItems.pages
+    insights = insightsPage === 1 ? insightItems.items : insights.concat(insightItems.items)
+  }
+
+  async function setDisplayingItems() {
+    const displayingItems = await queryExplorerItems({
```

Comment:
As @Woafflation described, separate insights' query was added as hotfix to support insights in the feed. Now it is not required, since it's natively supported by the api.
https://github.com/santiment/app/blob/b5b94dbc5d473a268d46ad76fa949fb10b6a9787/src/pages/Explorer/Category/ExplorerCategory.svelte#L91-L104
The inisght specific query function can be removed (LOC 82)